### PR TITLE
merge storyView mas com HUD mais ou menos

### PR DESCRIPTION
**Pull request by pietro1704**

### CODE REVIEW

File: challengeFinal/Assets/GameColors.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import Foundation
+import UIKit
+
+extension UIColor{
+    static let background = UIColor(named: "Background")!
+    static let buttonBackground = UIColor(named: "ButtonBackground")!
+    
+    //"red" conflicts with system
+    static let gameRed = UIColor(named: "Red")!
```
Comment (guilhermeShima) [inline challengeFinal/Assets/GameColors.swift]: You can name it `accentRead`?

File: challengeFinal/Flow/Story/View/StoryView.swift [hunk 2]
```diff
@@ -22,50 +22,116 @@ public class StoryView: UIView {
...
+    
+    private lazy var contentView: UIView = {
+        let content = UIView()
+        content.translatesAutoresizingMaskIntoConstraints = false
+        content.addSubview(textView)
+        content.addSubview(goToDecisionButton)
+        return content
     }()
 
+
```
Comment (guilhermeShima) [inline challengeFinal/Flow/Story/View/StoryView.swift]: Remove extra line

File: challengeFinal/Flow/Story/View/StoryView.swift [hunk 3]
```diff
@@ -22,50 +22,116 @@ public class StoryView: UIView {
...
 
     public init(with viewModel: StoryViewModel) {
         self.viewModel = viewModel
         super.init(frame: .zero)
-        backgroundColor = UIColor(red: 0.89, green: 0.88, blue: 0.85, alpha: 1.00)
+        backgroundColor = UIColor.background
         setupConstraints()
     }
-
+
```
Comment (guilhermeShima) [inline challengeFinal/Flow/Story/View/StoryView.swift:73]: Remove extra line

File: challengeFinalComponents/DecisionPointsView/DecisionPointsView.swift [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    
+    var pointsLabel: UILabel = {
+        let label = UILabel()
+        label.text = "0"
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    init(numberOfPoints: Int = 0, textSize: CGFloat = defaultLabelSize){
+
```
Comment (guilhermeShima) [inline challengeFinalComponents/DecisionPointsView/DecisionPointsView.swift]: Remove empty lines

File: challengeFinalComponents/ScrollableTextView.swift [hunk 5]
```diff
@@ -0,0 +1,42 @@
+//
+//  RegularTextView.swift
+//  challengeFinal
+//
+//  Created by Guilherme Domingues on 14/10/20.
+//
+
+import UIKit
+
+public class ScrollableTextView: UI {
```
Comment (guilhermeluisdomingues) [inline challengeFinalComponents/ScrollableTextView.swift]: 👀

**Review by guilhermeShima — APPROVED** [review]

File: challengeFinalTests/Components/DecisionPointsViewTests.swift [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+import XCTest
+@testable import challengeFinal
+import SnapshotTesting
+
+class DecisionPointsViewTests: XCTestCase {
+    func testView() {
+        let decisionView = DecisionPointsView()
+       
+        decisionView.backgroundColor = .blue
```
Comment (guilhermeluisdomingues) [inline challengeFinalTests/Components/DecisionPointsViewTests.swift]: remove color

**Comment by guilhermeShima** [comment]
nice job :)

**Review by guilhermeluisdomingues — CHANGES_REQUESTED** [review]

**Review by guilhermeluisdomingues — APPROVED** [review]

**Pull request merged**